packages/db/src/connect.ts

```diff
@@ -1,3 +1,6 @@
+import { logger } from "@truffle/db/logger";
+const debug = logger("db:connect");
```

Comment:
Are you adding this here for later?

Reply:
Yep - I added it to every module I could, in case we need it. We have a dedicated eslint rule to allow this as an unused variable. This is how we do things in other packages.